lib/ssl/tls13con.c

```diff
@@ -1611,6 +1611,13 @@ tls13_HandleCertificateRequest(sslSocket *ss, SSL3Opaque *b, PRUint32 length)
     if (rv != SECSuccess)
         goto loser;
 
+    /* We don't support post-handshake client auth, the certificate request
```

Comment:
I am surprised that this passes tests because we send a CR.context  of length 1.

http://searchfox.org/nss/source/lib/ssl/tls13con.c#1452